### PR TITLE
Add cy.deleteAllIndexes() in before block in test-api-key-required.cy.js

### LINKED ISSUE #334: deleteAllIndexes should also be added in test-api-key-required.cy file
In the `e2e` tests runned by Cypress, most of the files delete all indexes to reset the environment before testing.
The file `test-api-key-required.cy.js` does not delete all indexes in a `before` function.

Unfortunately, a test from that files requires that there are no indexes currently on the Meilisearch instance.

https://github.com/meilisearch/mini-dashboard/blob/5683d1a4d688ede3f4e938951b9e0ba4cd54f03f/cypress/e2e/test-api-key-required.cy.js#L40

To tackle that issue, we need to add a `before` method in that test file, running `deleteAllIndexes`. 

See example here:

https://github.com/meilisearch/mini-dashboard/blob/5683d1a4d688ede3f4e938951b9e0ba4cd54f03f/cypress/e2e/test-no-api-key-required.cy.js#L5

### PR DESCRIPTION
**Pull request by shyambh**
test-api-key-required.cy.js file

# Pull Request

## Related issue
Fixes #334

## What does this PR do?
- Add cy.deleteAllIndexes() in before block in the test file test-api-key-required.cy.js

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

**Comment by bidoubiwa** [comment]
Hello @shyambh,
Thank you very much for contributing to Meilisearch :heart:.
However, I am not available on the weekend, but I will be back on Monday :blush:.

*PS: This message was sent automatically!*

**Comment by shyambh** [comment]
Hi @bidoubiwa , I did exactly what was mentioned in the issue. However, I am not quite sure if this alone would fix the failing test 😅

File: cypress/e2e/test-api-key-required.cy.js [hunk 1]
```diff
@@ -4,6 +4,11 @@ const WRONG_APIKEY = Cypress.env('wrongApiKey')
 const WAITING_TIME = Cypress.env('waitingTime')
 
 describe(`Test API key required`, () => {
+
```
Comment (bidoubiwa) [inline cypress/e2e/test-api-key-required.cy.js:7]:
```suggestion
```

**Review by bidoubiwa — CHANGES_REQUESTED** [review]
A part the unecessary new-line, it looks good to me :)

**Comment by shyambh** [comment]
Hi @bidoubiwa , I have committed the suggested change removing the extra newline.

**Comment by mdubus** [comment]
Hello @shyambh 👋

First of all, I deeply apologize for the delay 🙈 

This looks perfect, so thanks a lot for your work here  🙏
(thanks also @bidoubiwa for your reviews)

bors merge

**Comment by shyambh** [comment]
Hi @mdubus , can you please also set the label for this as ```hacktoberfest-accepted``` ?

**Comment by mdubus** [comment]
Hello @shyambh 👋 

As the PR was made in a repository tagged with the `hacktoberfest` label, there's no need to add the `hacktoberfest-accepted` to your PR, it will count automatically for the Hacktoberfest challenge. The `hacktoberfest-accepted` label is only for specific cases, like a PR that we couldn't merge on time 😉

Have a nice day! ✨

**Comment by shyambh** [comment]
> 

Thank you very much 🙏🏼

**Pull request merged**